recipes/meshpy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "meshpy" %}
+{% set version = "2016.1" %}
+{% set sha256 = "a9e80c1f23f6a6cdc66f8f62ea63dc273664f357d25e572bc5edf5f9b22af7d9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt && cd test && python -m pytest
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - six
+  run:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - six
+
+test:
+  requires:
+    - pytest
+  imports:
+    - meshpy
+
+about:
+  home: http://github.com/inducer/meshpy
+  license: MIT and Proprietary
```

Comment:
[Triangle](https://www.cs.cmu.edu/~quake/triangle.html) and [TetGen](http://wias-berlin.de/software/tetgen/) are included in the package, but they both have obnoxious licenses. ("free as long as what you're doing is not commercial")

Both of those are non-essential. What's left at that point is an interface to the `gmsh` mesh generator. What do you suggest?